apps/src/templates/projects/PersonalProjectsTable.jsx

```diff
@@ -17,6 +17,8 @@ import DeleteProjectDialog from '@cdo/apps/templates/projects/deleteDialog/Delet
 import FrozenProjectInfoDialog from '@cdo/apps/templates/projects/frozenProjectInfoDialog/FrozenProjectInfoDialog';
 import {isSignedIn} from '@cdo/apps/templates/currentUserRedux';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {filter} from 'lodash';
```

Comment:
Just curious why you're using `filter` from 'lodash' vs JS native filter.

Reply:
Hmm, that's odd. I wasn't actually using it. :) 

I think it got added by VSCode automatically at some point as I was using auto-fill suggestions and I just missed that it happened. It was probably about to be caught by drone as a linting error. Good catch!